crates/compiler/src/plugin/attribute_macros/model.rs

```diff
@@ -56,16 +58,17 @@ impl DojoModel {
             &mut diagnostics,
         );
 
-        let model_name = struct_ast
+        let model_type = struct_ast
             .name(db)
             .as_syntax_node()
             .get_text(db)
             .trim()
             .to_string();
+        let model_name_snake = model_type.to_case(Case::Snake);
```

Comment:
```suggestion
        let model_type_snake = model_type.to_case(Case::Snake);
```
to be consistent with the change proposed here.